package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "openfisca.org",
+  "version": "1.0.0",
+  "description": "The public-facing website of OpenFisca for an international audience.",
+  "main": "publish.js",
+  "dependencies": {
+    "gh-pages": "^1.0.0"
+  },
+  "devDependencies": {
+    "gh-pages": "^1.0.0"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/openfisca/openfisca.org.git"
+  },
+  "author": "",
```

Comment:
Either fill this property in or remove it altogether (I would recommend to remove it in the current case).